scripts/test/slash.cjs

```diff
@@ -0,0 +1,15 @@
+/**
+ * The following code is copied from
+ * https://github.com/sindresorhus/slash/blob/98b618f5a3bfcb5dd374b204868818845b87bb2f/index.js
+ *
+ * MIT Licensed
+ * Author Sindre Sorhus @sindresorhus
+ */
+module.exports = function slash(path) {
```

Comment:
This one is intended to be used only with path input, however, `jest-serializer-path` use it to everything. It's working in most cases though.